mono/mini/mini-exceptions.c

```diff
@@ -240,11 +240,10 @@ static gboolean show_native_addresses = TRUE;
 static gboolean show_native_addresses = FALSE;
 #endif
 
-static _Unwind_Reason_Code
-build_stack_trace (struct _Unwind_Context *frame_ctx, void *state)
+static gboolean
+add_ptr_to_stack_trace (uintptr_t ip)
```

Comment:
It's because the value passed to this is a _Unwind_Ptr, which is just a typedef around uintptr_t.  http://clang.llvm.org/doxygen/unwind_8h.html#ab566a2905df286d9520b814e80f2c23f

I went with using the naked type because we want to treat them as char-stars later. I can change this if it's preferred to use another type stylistically, but this seemed the easiest to read and understand what we're actually getting.